development/release.md

```diff
@@ -12,11 +12,12 @@ The follow process should be followed for a FlowForge release.
 
 ## Steps
 
- - PR raised to update all relevent version numbers (Please reffer to [Packaging](packaging.md#package-verion-numbering) for details). This should include updating any entries in the `dependencies` section of the `package.json` if applicable and remember to run `npm install` and check in the new updated `package-lock.json` as part of the PR.
+ - Use the `checkout-release` script (from the admin project) to check out clean clones of the packages required to do a release 
+ (e.g. `./checkout-release 0.4.0` will create a directory called `release-0.4.0` containing all the required projects)
+ - From with in the `release-0.4.0` directory run the `prepare-release 0.4.0` script (from the admin project). This will raise update all the required
```

Comment:
```suggestion
 - From with in the `release-0.4.0` directory run the `prepare-release 0.4.0` script (from the admin project). This will update all the required
```